examples/aws-nodejs/public/index.html

```diff
@@ -13,6 +13,14 @@ <h2>AWS S3 multipart</h2>
     <div id="aws-multipart"></div>
     <script type="module">
       import { Uppy, Dashboard, AwsS3Multipart, AwsS3 } from "https://releases.transloadit.com/uppy/v3.9.1/uppy.min.mjs"
+      function serialize(data) {
```

Comment:
I'm not sure about this change. Why do we leave this up to the user? Can't we do this internally?